internal/worker_runner.go

```diff
@@ -242,6 +242,8 @@ func (wr *workerRunner) queryOBD() {
 				Name:      request.Name,
 				Value:     value,
 			})
+		} else if request.FormulaType() == "python" {
+			// todo: implement python formula decoding?
```

Comment:
@JamesReate , I am not sure as of now how to decode the Python formula; I need probably an example of a real Python formula type.

Reply:
for now we'll just have the autopi handle the decoding of the formula. 
If the formulatype is python, then the value will be returned as is from the json - let's review tomorrow to look for an example JSON response where the formula is applied by the autopi command.